i18n/fotema.ftl

```diff
@@ -0,0 +1,225 @@
+
```

Comment:
Separate `fotema.ftl.license` files are no longer needed for localisation files, so please would you add the SPDX licence tags at the top of the file? For this file format it should look like this:
```
# SPDX-FileCopyrightText: © 2024 David Bliss
#
# SPDX-License-Identifier: GPL-3.0-or-later
```